src/test/java/foodwhere/model/common/NameTest.java

```diff
@@ -1,4 +1,4 @@
-package foodwhere.model.stall;
+package foodwhere.model.common;
```

Comment:
Wondering why this is model.common instead of model.commons

Reply:
Fixed in latest commit.